webapp/packages/core-blocks/src/ObjectPropertyInfo/ObjectPropertyInfoForm/RenderField.tsx

```diff
@@ -35,7 +32,26 @@ interface RenderFieldProps {
   className?: string;
 }
 
-const RenderField: React.FC<RenderFieldProps> = observer(function RenderField({
+function isCheckbox(property: ObjectPropertyInfo) {
+  return property.dataType?.toLowerCase() === 'boolean';
+}
+
+function getDefaultValueFor(property: ObjectPropertyInfo) {
+  const checkbox = isCheckbox(property);
+  const value = property.value;
+
+  if (value === null || value === undefined) {
+    return checkbox ? false : '';
+  }
+
+  if (typeof value === 'string') {
+    return checkbox ? value === 'true' : value;
+  }
+
+  return value.displayName || value.value || JSON.stringify(value);
+}
```

Comment:
function behavior is not clear because of `defaultValue` property in `ObjectPropertyInfo` object